.changeset/gentle-workflows-authorize.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/code-sdk': minor
+---
+
+Add a `createSaveWorkflowTool` factory with an optional deny-only authorization hook that receives the native request context and a detached workflow definition.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)